### PR TITLE
feature/streamline logging and exception messages

### PR DESCRIPTION
**Pull request by chr-ber**
The most important aspect of this pull request is the addition of a missing string interpolation symbol to one of the log messages, found in the first commit -> https://github.com/dotnet/nuget-trends/commit/c50e6dafe343779b6537cc6a1a906ae052bd7fa3. I encountered this issue when running the project locally.

While I was at it, I also reviewed all log and exception messages in the project and unified their style. I understand if that seems a bit unnecessary; I guess I got carried away. 😅 I believe the first commit should be accepted at the very least. If the rest ends up in the 🗑️. No harm done!

In general, I tried to apply these simple rules:

- All dynamic variables must be wrapped in single quotes.
- Static variables should not be wrapped.
- All messages must end with punctuation.

### CODE REVIEW

File: src/NuGetTrends.Data/BasePostgresContext.cs [hunk 1]
```diff
@@ -1,8 +1,8 @@
+using System.Text.RegularExpressions;
```
Comment (bruno-garcia) [inline src/NuGetTrends.Data/BasePostgresContext.cs]: doesn't seem needed?

```suggestion
```

**Review by bruno-garcia — COMMENTED** [review]
Thanks Chris! only 1 nitpick otherwise LGTM

File: src/NuGetTrends.Data/BasePostgresContext.cs (hunk 1 above)
Comment (cxbrgr) [reply src/NuGetTrends.Data/BasePostgresContext.cs]: removed from the commit

**Comment by bruno-garcia** [comment]
Thanks again Chris!

**Pull request merged**